action-templates/actions/action-maven-release/action.yml

```
@@ -69,14 +73,23 @@ runs:
       id: maven-release-step
       shell: bash
       run: |
+        # configure git
         git config --global user.email "github-actions@github.com"
         git config --global user.name "GitHub Actions"
+        # get maven artifact id and provide as output
         MVN_ARTIFACT_ID=$(mvn -f .${{inputs.app-path}}/pom.xml org.apache.maven.plugins:maven-help-plugin:3.2.0:evaluate -Dexpression=project.artifactId -q -DforceStdout)
         echo $MVN_ARTIFACT_ID
         echo "MVN_ARTIFACT_ID=$MVN_ARTIFACT_ID" >> $GITHUB_OUTPUT
-        mvn release:prepare release:perform -f ./${{inputs.app-path}}/pom.xml -B -DreleaseVersion=${{ inputs.releaseVersion }} -DdevelopmentVersion=${{ inputs.developmentVersion }} -Darguments="-Dmaven.deploy.skip=${{ inputs.skipDeployment }}"
+        # construct maven release arguments
+        if [[ "${{ inputs.use-pr }}" == "true" ]]; then
+          mvn_arguments="-Dmaven.deploy.skip=${{ inputs.skipDeployment }} -DskipCommit=true -DskipPush=true"
```

```suggestion
          mvn_arguments="-Dmaven.deploy.skip=${{ inputs.skipDeployment }} -DskipCommit=${{ inputs.use-pr }}-DskipPush=${{ inputs.use-pr }}"
```

kannst du ja gleich so schreiben dann brauchst kein if

---

Achtung da fehlt noch ein Leerzeichen zwischen den Argumenten.

---

Done